### DISCUSSION TITLE
Failed status after scanned QR Code

### DISCUSSION THREAD

**Discussion by JoshuaBarawa** [devlikeapro/waha#397] [Q&A]
I keep getting failed status after scanning the QR code. Any help?

**Comment by willpexeroo** [comment]
my session status is STARTING and dont change to get a QRCODE

**Comment by MVidal80** [comment]
same here

**Comment by joaopintof** [comment]
the same here :-(

[![patron:PLUS](https://img.shields.io/badge/patron-PLUS-a0e6ba)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Reply by devlikepro** [reply]
Hi! Make sure you're using the latest version - `devlikeapro/waha-plus` (or `waha` docker image), it's been renamed
- https://boosty.to/wa-http-api/posts/6a58df1a-6e47-4ed2-a2b0-b673e99f2070
- https://boosty.to/wa-http-api/posts/a6c01a2d-704e-4066-978a-890e84de9da1

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
Hi! Make sure you're using the latest version - `devlikeapro/waha-plus` (or `waha` docker image), it's been renamed
- https://boosty.to/wa-http-api/posts/6a58df1a-6e47-4ed2-a2b0-b673e99f2070
- https://boosty.to/wa-http-api/posts/a6c01a2d-704e-4066-978a-890e84de9da1

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
